sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -120,6 +122,13 @@ const createHTLC = async (
     const assetExchangeAgreementStr = createAssetExchangeAgreementSerialized(assetType, assetID, recipientECert, "");
     const lockInfoStr = createAssetLockInfoSerialized(hash, expiryTimeSecs);
 
+    //If timeoutCallback is not defined, start automated listener
+    if (typeof timeoutCallback !== "function") 
+    {
+        //TODO: Need to acquire lockerECert
+        StartHTLCAssetLockListener(contract, contract.chaincodeId, assetType, assetID, recipientECert, "", assetLockCallbackTimed);
```

Comment:
I don't see `assetLockCallbackTimed` defined earlier in this function, either as a parameter or as a local variable. Also, why is it called `Timed`?